ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/432__data-science-pipelines-tekton.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline using kfp_tekton python package
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/Pipelines/OpenShiftPipelines.robot
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Library             ../../../../libs/DataSciencePipelinesKfpTekton.py
+Suite Setup         Data Science Pipelines Suite Setup
+Suite Teardown      RHOSi Teardown
+
+
+*** Variables ***
+${DATA_SCIENCE_PIPELINES_APPLICATION_YAML}=    ods_ci/tests/Resources/Files/data-science-pipelines-sample.yaml
+${URL_TEST_PIPELINE_RUN_YAML}=                 https://raw.githubusercontent.com/opendatahub-io/data-science-pipelines-operator/main/tests/resources/dsp-operator/test-pipeline-run.yaml
+
+
+*** Test Cases ***
+Verify ODS users can create and run a data science pipeline using the kfp_tekton python package
+    [Documentation]    Creates, runs pipelines with regular user. Double check the pipeline result and clean
+    ...    the pipeline resources.
+    [Tags]      Sanity
+    ...         Tier1
+    ...         ????
+    End To End Pipeline Workflow Using Kfp_tekton    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    pipelineskfptekton1
+
+
+*** Keywords ***
+End To End Pipeline Workflow Using Kfp_tekton
+    [Documentation]    Create, run and double check the pipeline result using Kfp_tekton python package. In the end,
+    ...    clean the pipeline resources.
+    [Arguments]     ${username}    ${password}    ${project}
+    Remove Pipeline Project    ${project}
```

Comment:
Same as earlier use openshift library

Reply:
Remove Pipeline Project will be addressed in other PR. The current code [double check ](https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/libs/DataSciencePipelinesAPI.py#L84-L97) wait because it could be in Terminating status